src/components/shared/Swiper.tsx

```diff
@@ -0,0 +1,329 @@
+/**
+ * Swiper
+ * Renders a swipable set of screens passed as children and pagination indicators.
+ * Adapted from https://rationalappdev.com/complete-guide-to-mobile-app-onboarding-with-react-native/
+ */
+
+import React, { Component } from "react";
+import {
+  Dimensions,
+  NativeSyntheticEvent,
+  NativeScrollEvent,
+  Platform,
+  ScrollView,
+  ScrollViewProps,
+  StyleSheet,
+  View,
+  ViewPagerAndroidOnPageScrollEventData
+} from "react-native";
+import RoundedButton from "./RoundedButton";
+
+// Detect screen width and height
+const { width, height } = Dimensions.get("window");
+
+type SwiperProps = {
```

Comment:
I don't know what a few of these actually do, some  comments would be nice if we do keep all this configurability.

Reply:
Will do!